web/src/js/ducks/ui/flow.js

```diff
@@ -94,15 +94,6 @@ export function updateEdit(update) {
 export function stopEdit(flow, modified_flow) {
     let diff = getDiff(flow, modified_flow)
     return (dispatch) => {
-        if (diff.response && diff.response.content) {
-            dispatch(flowsActions.updateContent(flow, diff.response.content, "response"))
-            delete diff.response.content
-        }
-        if (diff.request && diff.request.content) {
-            dispatch(flowsActions.updateContent(flow, diff.request.content, "request"))
```

Comment:
we should remove this line right ? because the sever should tell the client that the flow is changed ?  but thats not working :( i try to find the bug, but i did not found it